### DISCUSSION TITLE
Open Community Infra Program

### DISCUSSION THREAD

**Discussion by fewensa** [ringdao-community-guild/discussion#133] [General]
## Intro

After 3 quarters of operation of Community RPC, it has been successfully demonstrated that the community provides better assistance to the Darwinia organization.

During this period, it successfully operated an RPC service provider that was completely driven by the community, providing complete and stable services, especially Crab RPC, which provided almost more than 80% of the entire load.

Here, we plan to upgrade the Community RPC service to the Open Community Infra Program to achieve the following two purposes

- Optimize the use of funds
- Provide more reliable infrastructure services

## Works

The main work is to improve and enhance existing infrastructure and develop and support new infrastructure services.

Services supported by the Open Community Infra Program include but are not limited to  

- Self-developed  
- Use third-party services
- Support development-related expenses

## Budget

Open Community Infra Program will apply for 3000U worth of RING as rewards per quarter. 
Funds will be applied to DCDAO every quarter, and the funds will be deposited in a special multi-signature account (safe) on the Ethereum chain.
https://app.safe.global/home?safe=eth:0xDede81390194E5173997429C2cD8515c65be02E3

Funds will be distributed according to the following principles:
- Necessary distribution, such as expenses required for programs that have been run, and rewards for personnel who run services.
- Any expenditures to enhance infrastructure, such as using third-party services to support development
- Expenditures allowed by fund multi-signature management members or a majority of DCDAO members (corresponding proposals need to be submitted)

## Reference

- [Request: Will there be some community RPC services support Darwinia2 & Crab2?](https://github.com/orgs/dcdao/discussions/11)
- [Community RPC Experimental Program](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x26be609917096095e8fb2a617d6def3fa62b3240285a9887bcd9184afb2a0ddd)
- [Upgrade of the Community RPC Experimental Program and Expansion to the Community Infrastructure Program](https://github.com/orgs/dcdao/discussions/132)

**Comment by fewensa** [comment]
Community RPC reward distribution strategy change plan
=== 

Such as Q4 of 2023 mentioned, The rewards strategy will be change.
The new strategy is Q1 of 2024 every node will reward 100 USD per month, Since Q2 of 2024, adjust the incentives for Community RPC nodes to a maximum of 50 USD per node per month.

- [Community RPC Reward Distribution at Q1 of 2024](https://github.com/orgs/dcdao/discussions/130)
- [Community RPC Reward Distribution at Q4 of 2023](https://github.com/orgs/dcdao/discussions/60)
- [Upgrade of the Community RPC Experimental Program and Expansion to the Community Infrastructure Program](https://github.com/orgs/dcdao/discussions/132)

**Comment by fewensa** [comment]
The first stage, Work to be done includes https://github.com/orgs/dcdao/discussions/132

**Comment by fewensa** [comment]
@wuminzhe @WoeOm are Community RPC node runner,  please leave your's ethereum address, I'll create a new safe address to receive RING.

**Reply by WoeOm** [reply]
0x6508CF6ff0bDf6865F8760CfB007edC2eCfBE5bC  woeom

**Reply by wuminzhe** [reply]
0xA4bE619E8C0E3889f5fA28bb0393A4862Cad35ad aki

**Reply by fewensa** [reply]
0x3198D3070D84cB6798651AA3c02828Ba5866b296 fewensa

**Comment by fewensa** [comment]
https://app.safe.global/home?safe=eth:0xDede81390194E5173997429C2cD8515c65be02E3
